pom.xml

```diff
@@ -89,22 +89,56 @@ under the License.
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <sourceReleaseAssemblyDescriptor>source-release</sourceReleaseAssemblyDescriptor>
     <gpg.useagent>true</gpg.useagent>
-    <minimalMavenBuildVersion>3.2.5</minimalMavenBuildVersion>
+    <minimalMavenBuildVersion>3.6.3</minimalMavenBuildVersion>
```

Comment:
We have separate issue for it: https://issues.apache.org/jira/browse/MPOM-399

Reply:
I can back that out of here.